stepactions/eaas-create-ephemeral-cluster-hypershift-aws/0.1/eaas-create-ephemeral-cluster-hypershift-aws.yaml

```diff
@@ -71,6 +79,18 @@ spec:
     #!/bin/bash
     set -eo pipefail
 
+    # Resolve imageContentSources from file (if provided) or fallback to inline string
+    ICS_VALUE=""
+    if [[ -n "$IMAGE_CONTENT_SOURCES_FILE" && -f "$IMAGE_CONTENT_SOURCES_FILE" ]]; then
```

Comment:
I think we should also check for the case where the ICS file param is provided but the file doesn't exist. Right now this will silently continue without configuring the ICS. Users _may_ only notice after waiting for the cluster to provision. I think we should fail instead.